lib/message.rb

```diff
@@ -0,0 +1,14 @@
+require './lib/morse_translator'
+
+class Message
+  attr_reader :text, :translator
+
+   def initialize(text, translator)
+     @text = text
+     @translator = translator
+   end
+
+   def translation
```

Comment:
This looks like a great start!  I think you are probably on the right track, just didn't quite get to implementing this method.